src/views/GameSettingsWizard/ActionsStep/PickConsumptions/__tests__/PickConsumptions.test.tsx

```
@@ -63,16 +65,29 @@ vi.mock('@/components/GameForm/YesNoSwitch', () => ({
-    <div data-testid={`incremental-select-${option}`}>
-      <label>{option}</label>
-      <input
-        type="number"
-        onChange={(e) => onChange({ target: { value: parseInt(e.target.value) } })}
-        data-testid={`intensity-input-${option}`}
-      />
+// Mock MultiSelectIntensity component
+vi.mock('@/components/MultiSelectIntensity', () => ({
+  default: ({ actionName, selectedLevels, onChange }: any) => (
+    <div data-testid={`multi-select-intensity-${actionName}`}>
+      <label>{actionName}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)